guides/common/modules/proc_configuring-for-uefi-http-boot-ipv6-provisioning.adoc

```diff
@@ -25,5 +25,5 @@ ifndef::foreman-deb[]
 ----
 endif::[]
 ifdef::katello,orcharhino,satellite[]
-. Synchronize the {EL} 8 kickstart repository.
+. Synchronize the {EL} kickstart repository.
```

Comment:
This is unrelated because it's for the host to be provisioned. You can argue this doesn't make sense in this file, because it's not related to configuring the `{Project}` but I wouldn't fix it as part of this PR.

Reply:
I can cover here :+1: